OpenSim/Actuators/Schutte1993Muscle_Deprecated.h

```diff
@@ -102,7 +102,13 @@ OpenSim_DECLARE_CONCRETE_OBJECT(Schutte1993Muscle_Deprecated,
     virtual bool setActivation1(double aActivation1);
     virtual bool setActivation2(double aActivation2);
     virtual bool setDamping(double aDamping);
-
+    
+    //TODO: Is this the right place to put this method?
```

Comment:
@thomasklau do you mean if this method should go in the protected or private section?